### ISSUE TITLE
Reproducing results reported in paper

### ISSUE THREAD

**Issue by Francis777** [lty2226262/Car_studio#2]
Hi, 
Congrats on getting this great work accpeted and thanks for releasing the code! I'm wondering which experiment config leads to the result shown in your TABLE II(singleview supervision zero-shot learning)? I tried your default experiment(`car-nerf-km-sv-with-mask`), the result is ok but has quite a bit of gap(PSNR <15 in this experiment, compared with >20 as in the paper). 
![image](https://github.com/lty2226262/Car_studio/assets/25652281/c4f7cf05-41b4-4822-86f4-5160bc77cadc)

**Comment by lty2226262** [comment]
Thank you for your interest in our work. We sincerely apologize for any confusion caused. We would like to clarify that the default configuration provided is primarily intended for demonstrating functionality and creating demos.

To replicate the experiments described in our paper, we recommend utilizing the kitti-det dataset (referred to as kitti-obj in the readme file) and increasing the max_num_iterations parameter by 10 times, setting it to 500,000 to achieve more accurate results.

If your objective is to improve the quality of the output, tuning a larger iteration step will also be effective.

**Comment by Francis777** [comment]
Hi @lty2226262, thanks for your reply! Following your suggestion I tried using both `km`+`ko` and increased training steps to 500k, the result does look slightly better visually, however the metrics are still far from the ones reported in the paper. I wonder if there're other hyperparameters that are critical to the result? Also would it be possible to release the model checkpoint?

**Comment by lty2226262** [comment]
Sure, I will review the hyperparameters in the log and provide you with a specific configuration file. If you require the checkpoint, I will also send it to you. However, I have encountered some issues with resuming from a checkpoint and evaluating it, and I am currently uncertain about how to resolve these bugs.

**Comment by Francis777** [comment]
That'd be very helpful, thanks Tianyu! As we have our own implementation based on the paper before your code release, we're keen to compare and understand the differences. 
Here're some questions I have for now: 
- do you find setting `n_instance_per_batch` to >1 important? 
- what's the difference between `CarNeRF` and `CarNeRFSymm`?  
- what's the difference between stage 1 and stage 2? 
- If you pass the car masks in `mask_filenames` in the dataparser outputs, the pixel sampler(`car_studio/data/custom_pixel_samplers.py`) will only sample pixels within the mask, which means the background pixels will never be sampled, but in the meantime you have a segmentation loss "to prevent our model from learning the background color or shapes", I find this combination a bit confusing, maybe I'm missing something here? 

Thanks in advance!
